packages/docs/src/examples/v-menu/misc-transition.vue

```diff
@@ -1,68 +1,35 @@
 <template>
-  <div class="d-flex justify-space-around">
-    <v-menu
-      transition="scale-transition"
-    >
+  <div class="d-flex justify-space-around flex-wrap ga-1">
+    <v-menu transition="scale-transition">
       <template v-slot:activator="{ props }">
```

Comment:
If we're already here, should update props to look like this:

```html
<template v-slot:activator="{ props: activatorProps }">
```

Since that's how we recommend users do it.